test/integration/reconnect/docker.js

```diff
@@ -0,0 +1,29 @@
+const _       = require('lodash');
+const $Docker = require('dockerode');
+
+function Docker() {
+  this.dockerAPI = new $Docker({ socketPath: '/var/run/docker.sock' });
```

Comment:
This isn't probably windows compatible. Is there way to use some other communication method that is not unix domain socket?